db/migrate/20220120201803_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :postcounter
+
```

Comment:
- [ ]  Optional suggestion

- I would recommend renaming/changing the columns like `posts_counter`